### PR TITLE
Feat/component test

### PR DESCRIPTION
**Pull request by AlissonRoss**
Allows the user to dynamically send over tailwindcss classes to change the button's color or the text. These are sent over as props. There are also predefined ones in-case the user does not fill all of them out.

Firefox on MacOS
image

Screenshot 2024-08-26 at 2 05 50 PM

### CODE REVIEW

File: src/app/page.js
Comment (missile720) [inline src/app/page.js:1]: There is old default vercel code with the main tag. It was a div tag with no class names in it

**Review by missile720 — CHANGES_REQUESTED** [review]
The classNames on the main tag are from the original vercel default.

**Comment by AlissonRoss** [comment]
> The classNames on the main tag are from the original vercel default.

It's fixed now. Ty

**Review by missile720 — APPROVED** [review]
Looks good!

**Review by davidxiong117 — APPROVED** [review]
Looks good! Holy smokes.

**Pull request merged**